docs/criteria/accelerator-stages.md

```diff
@@ -9,51 +9,41 @@ Current supported data accelerators (DuckDB, Postgres, SQlite, Arrow) release st
 ### Feature set complete
 
 - [ ] Streaming when accelerating/loading
-- [ ] Index
+- [ ] Streaming when querying
+- [ ] Supports primary keys and indexes
+- [ ] Supports federation in a single dataset
+- [ ] Supports push-down federation across multiple datasets within the same accelerator
 - [ ] On conflict behaviors - Drop/Upsert
-- [ ] File mode for SQLite and DuckDB
+- [ ] File mode for supported accelerator engines, i.e. SQLite, DuckDB, etc.
 - [ ] Common/core arrow data type supports: Null, all Int types, all Float types, Time32, Time64, Timestamp (with/without TZ), Date32, Date64, Duration, Interval, Binary/LargeBinary/FixedSizeBinary, Utf8/LargeUtf8, List/FixedSizeList/LargeList, Struct, Decimal128/Decimal256
 - [ ] No known major bugs/issues
 
 ### Test Coverage
```

Comment:
Should there be a statement regarding the use of indexes in accelerators when running benchmarks?